### ISSUE TITLE
Generates invalid config for `engine_version`

### ISSUE THREAD

**Issue by pallaswept** [benjamimgois/goverlay#123]
GOverlay 1.1.1 stores the `engine_version` option with the invalid name `engine`. This generates an error: 
```
> mangohud vkcube  
[2024-07-01 13:49:29.185] [MANGOHUD] [error] [overlay_params.cpp:647] Unknown option 'engine'
```
I don't know this codebase well, but a simple replacement of this string in a few places seems like it should sort it out. I'd be happy to file a PR if that would help?

**Comment by benjamimgois** [comment]
Thanks for reporting @pallaswept , will fix it on upstream.

**Comment by benjamimgois** [comment]
fixed in 1.2 release

**Comment by pallaswept** [comment]
Thanks very much!
Wow, congrats on that release, that's a huge one!!
